metricbeat/module/mongodb/dbstats/data.go

```diff
@@ -0,0 +1,34 @@
+package dbstats
+
+import (
+	s "github.com/elastic/beats/metricbeat/schema"
+	c "github.com/elastic/beats/metricbeat/schema/mapstriface"
+)
+
+var schema = s.Schema{
+	"db":           c.Str("db"),
+	"collections":  c.Int("collections"),
+	"objects":      c.Int("objects"),
+	"avg_obj_size": c.Int("avgObjSize"),
+	"data_size":    c.Int("dataSize"),
+	"storage_size": c.Int("storageSize"),
+	"num_extents":  c.Int("numExtents"),
+	"indexes":      c.Int("indexes"),
+	"index_size":   c.Int("indexSize"),
+	// mmapv1 only
+	"ns_size_mb": c.Int("nsSizeMB"),
```

Comment:
Should this field be optional as based on the comment, I assume it does not always exist?